lib/debug/server_dap.rb

```diff
@@ -621,8 +621,11 @@ def process_dap args
 
       case type
       when :backtrace
+        levels = req.dig('arguments', 'levels') || @target_frames.length
+        startFrame = req.dig('arguments', 'startFrame') || 0
```

Comment:
Given the usage below, I think this part should be:

```rb
start_frame = req.dig('arguments', 'startFrame') || 0
levels = req.dig('arguments', 'levels') 

if levels && levels != 0
  end_frame = start_frame + levels
  end_frame = -1 if end_frame > @target_frames.length
else
  end_frame = -1
end
```

Because `levels` isn't an absolute location, just a number for the next page.